src/main/php/org/bovigo/vfs/vfsStreamAbstractContent.php

```diff
@@ -148,7 +148,7 @@ public function getType()
      * sets the last modification time of the stream content
      *
      * @param   int  $filemtime
-     * @return  vfsStreamContent
+     * @return  $this
```

Comment:
Who uses documentation produced by `phpdoc`? I don't. But me myself and many many my collegues uses IDE and it comes very important to have expected behaviour. @mikey179 which IDE you use?